.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+name: Test with Code Coverage
+
+jobs:
+  test:
+    name: Test
+    env:
+      PROJECT_NAME_UNDERSCORE: rust_ci_github_actions_workflow
+      CARGO_INCREMENTAL: 0
+      RUSTFLAGS: -Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off -Zpanic_abort_tests -Cpanic=abort
+      RUSTDOCFLAGS: -Cpanic=abort
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
```

Comment:
I don't understand a lot from this ci, could you give a reference to documentation in answer, please? For example, I don't understand, why you need nightly toolchain.